src/main/java/domain/ArithmeticCalculator.java

```diff
@@ -0,0 +1,44 @@
+package domain;
+
+import java.util.List;
+
+public class ArithmeticCalculator {
+
+    private static final int INITIAL_RESULT_NUM = 0;
+    private static final int INITIAL_NUMBER = 1;
+
+    private final List<Integer> numbers;
+
+    public ArithmeticCalculator(List<Integer> numbers) {
+        this.numbers = numbers;
+    }
+
+    public int plusCalculate() {
+        int result = INITIAL_RESULT_NUM;
+        for (Integer number : numbers) {
+            result += number;
+        }
+        return result;
+    }
+
+    public int minusCalculate() {
+        return numbers.stream().reduce(INITIAL_RESULT_NUM,
```

Comment:
오 스트림까지~~
객체지향 생활체조 4번 따르면 개행 이렇게 바뀌겠네요!

```suggestion
        return numbers.stream()
                       .reduce(INITIAL_RESULT_NUM,(result, number) -> result - number);
```